content/material/2019-10-29-1st-rhus-meeting/2019-10-29-1st-rhus-meeting.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "First Rhus meeting"
+author: "Stefan McKinnon Høj-Edwards <smhe@kamstrup.dk>"
+date: "29th oct 2019"
+output: 
+  ioslides_presentation: 
+    logo: Rhus logo.png
```

Comment:
The output isn't necessary, since blogdown automatically generates the appropriate output. Do you want to add any categories or tags? Like in the other Rmd files?

Reply:
It is the slides from the meeting. How do you suggest we handle 1 to multiple slides/supplemental files per meeting?

Reply:
Great question. We could move the slides into a new folder in `static` called `slides`. Then create one material Rmd file and link the slides inside that file? Let me think about that more.

Maybe for now make the other changes and I'll merge in and finding out what might work best.

Reply:
I'll leave the output format in, until we resolve how to deal with slides - this page seems to render nicely as an ordinary page.